packages/pwa-kit-runtime/src/ssr/server/build-remote-server.test.js

```diff
@@ -30,3 +72,16 @@ describe('remote server factory test coverage', () => {
         expect(endpoint).toBeDefined()
     })
 })
+
+describe('extensions', () => {
+    test('can register extensions properly via _setupExtensions', () => {
+        const app = RemoteServerFactory._createApp(opts())
+        expect(app.__extensions).toBeDefined()
+        return request(app)
+            .get('/test-extension')
+            .expect(200)
+            .then((res) => {
+                expect(res.text).toBe('test')
+            })
```

Comment:
I guess we can iterate here and add cover the scenario for more than one extension.

```
            })
            .then(() => {
                return request(app)
                    .get('/another-extension')
                    .expect(200)
                    .then((res) => {
                        expect(res.text).toBe('another test')
                    })
            })
```

Reply:
great idea. done!